ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -117,13 +117,32 @@ is_loadbalancing_enabled() {
 	return 0
 }
 
+get_user_agent(){
+	local userAgentInfos gluon_version board_name openwrt_version kernel fw_release user_agent
```

Comment:
As we want to migrate this script to Lua in the long run, should we maybe move the function to the lua code?

Reply:
so that lua code only returns the UserAgent?
i orientatet me on this:
T0biii/community-packages@patch-2/ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink#L193-L197

but i think it is maybe easier to get only the User Agent

Reply:
that removed a ton of code :) perfect